synapse/tests/test_cortex.py

```diff
@@ -1303,9 +1303,6 @@ def onAdd(node):
 
         async with self.getTestCoreAndProxy() as (realcore, core):
 
-            msgs = await alist(core.storm('|help'))
```

Comment:
is this now allowed? or is this now bad syntax?

Reply:
this is now bad syntax.

Reply:
we should add this as a test to throw exception somewhere if we don't have it already

Reply:
Can we put this back after the previous discussion and tweaks?